wcag-ruleset-review-process.md

```diff
@@ -45,7 +45,7 @@ A facilitator of the ACT Task Force creates a survey for the rule, in which Task
 
 This survey will be open for at least 5 work days, to give ACT Task Force members sufficient time to review the rule. No more than 5 rule surveys will be due in the same week.
 
-If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+If the rule does not pass this step, the liaison has 5 work days to put the feedback in a new issue for the proposal using the template in appendix 1. They must then close the issue. When submitting the feedback, the liaison must notify the person who submitted the rule and ask them when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```

Comment:
```suggestion
If the rule does not pass this step, the liaison has 5 work days to put the feedback in a new issue for the proposal using the template in appendix 1. They must then reference the new issue in the "New rule" issue and close the "New rule" issue. When submitting the feedback, the liaison must notify the person who submitted the rule and ask them when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```

Reply:
@wilco We should probably make the "appendix 1" in bullet 1 above a link to appendix 1. Can you do that please?